src/common/low_precision_transformations/src/markup_precisions.cpp

```diff
@@ -152,6 +152,7 @@ bool ov::pass::low_precision::MarkupPrecisions::isPrecisionPreserved(const std::
         { name<opset1::Relu>() },
         // TODO: there are conditions
         { name<opset2::BatchToSpace>() },
+        { name<opset1::Broadcast>() },
```

Comment:
Should we support `v3::Broadcast` as well?

Reply:
thanks, done